ratis-proto/src/main/proto/Raft.proto

```diff
@@ -73,6 +73,7 @@ message StateMachineLogEntryProto {
   enum Type {
     WRITE = 0;
     DATASTREAM = 1;
+    FORWARD = 2;
```

Comment:
It seems not needed since StateMachine only sees DATASTREAM requests but not FORWARD requests.